### PR TITLE
[Documentation] Customizing Sylius via events vs. state machines

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q               | A
| --------------- | ---
| Doc fix?        | no |
| New docs?    | yes |
| Related tickets |  |
| License         | MIT |

### CODE REVIEW

File: docs/book/architecture/events.rst [hunk 1]
```diff
@@ -70,6 +70,18 @@ The events already used in Sylius are described in the Book alongside the concep
...
         $ php bin/console debug:event-dispatcher | grep sylius
 
+Customizations
+--------------
+
+.. note::
+
+    **Customizing logic via Events vs. State Machines**
+
+    The logic in which Sylius operates can be customized in tow ways. First of them is using the state machines: what is
```
Comment (venyii) [inline docs/book/architecture/events.rst]: tow -> two
what is ... -> this is ...

File: docs/book/architecture/events.rst [hunk 2]
```diff
@@ -70,6 +70,18 @@ The events already used in Sylius are described in the Book alongside the concep
...
+    really useful when you need to modify business logic for instance modify the flow of checkout,
```
Comment (venyii) [inline docs/book/architecture/events.rst]: of **the** checkout

File: docs/book/architecture/events.rst [hunk 3]
```diff
@@ -70,6 +70,18 @@ The events already used in Sylius are described in the Book alongside the concep
...
+    and the second is listening on the kernel events related to the entities, which is helpful for modifying the HTTP responses
+    visible directly for the user, like displaying notifications, sending emails.
```
Comment (venyii) [inline docs/book/architecture/events.rst]: visible directly for -> visible directly to

**Comment by pjedrzejewski** [comment]
Thanks Magda!

**Pull request merged**